KOService.Application/Commands/Employee/UpdateEmployeeCommand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using KOService.Domain.Authentication;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace KOService.Application.Commands.Employee
+{
+    public class UpdateEmployeeCommand : IRequest
+    {
+        public string Id { get; set; }
```

Comment:
Tak z ciekawości, jest jakiś argument przeciw używaniu tu Guida zamiast stringa (w ID?) Ja używam typu Guid i sie mapuje dobrze.

Reply:
no jak się mapuje dobrze to nawet lepiej użyć guida nawet. Nie zastanawiałem się nad tym w sumie, jakoś tak użyłem stringa.